lib/brains/tests/samples/customer-error.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<api-error-response>
+  <errors>
+    <errors type="array"/>
+    <customer>
+      <errors type="array">
+        <error>
+          <code>91610</code>
+          <attribute type="symbol">id</attribute>
+          <message>Customer ID is invalid (use only letters, numbers, '-', and '_').</message>
+        </error>
+      </errors>
+    </customer>
+  </errors>
+  <params>
+    <customer>
+      <id>sdfsdf:sdf:</id>
+    </customer>
+  </params>
+  <message>Customer ID is invalid (use only letters, numbers, '-', and '_').</message>
+</api-error-response>
```

Comment:
Also, if this XML changes but the library interface stays the same, the person updating this code will be sad